code/modules/reagents/reagent_containers/food/drinks/bottle.dm

```diff
@@ -368,8 +368,8 @@
 	reagents.add_reagent(/datum/reagent/drink/space_cola, 100)
 
 /obj/item/reagent_containers/food/drinks/bottle/space_up
-	name = "\improper Space-Up"
-	desc = "Tastes like a hull breach in your mouth."
+	name = "\improper Mix-Up"
+	desc = "a delightful fusion of flavors that offers a unique and invigorating experience."
```

Comment:
Can't you fucking decide on one name instead of naming same reagent differently each time?

Reply:
I didn't originally call it Space-Up, did I.